assets/tests/components/detours/detourMap.test.tsx

```diff
@@ -47,19 +47,15 @@ const DetourMapWithDefaults = (
 describe("DetourMap", () => {
   test("when `originalShape` is clicked, fires `onClickOriginalShape`", async () => {
     const onClickOriginalShape = jest.fn()
-    const shapePoint = { lat: 0, lon: 0 }
     const { container } = render(
-      <DetourMapWithDefaults
-        originalShape={[shapePoint]}
-        onClickOriginalShape={onClickOriginalShape}
-      />
+      <DetourMapWithDefaults onClickOriginalShape={onClickOriginalShape} />
     )
 
     fireEvent.click(
       container.querySelector(".c-detour_map--original-route-shape")!
     )
 
-    expect(onClickOriginalShape).toHaveBeenNthCalledWith(1, shapePoint)
+    expect(onClickOriginalShape).toHaveBeenCalled()
```

Comment:
This test was giving me some grief. I have a mild preference for being able to test more directly that the point that was actually clicked is the point passed into `onClickOriginalShape`, but I can't figure out how to simulate what point was clicked. (Passing `{ latlng: {lat: 0, lng: 0} }` as a second argument to `fireEvent` didn't seem to work ☹️)